### DISCUSSION TITLE
关于Go Crypto

### DISCUSSION THREAD

**Discussion by Trisia** [emmansun/gmsm#68] [Ideas]
您好，请问您是否有有意向Go官方库提交关于SM系列算法的PR？

这个是我的个人想法。

谢谢作者

**Comment by emmansun** [comment]
我看过有关golang 支持SM系列算法的提议，被拒绝了。

- [proposal: x/crypto/sm3: new package](https://github.com/golang/go/issues/24898)
- [proposal: crypto: support bn, nums, sm2 curves](https://github.com/golang/go/issues/52778)
